### PR TITLE
Advance by the correct number of position even for unknown characters

### LINKED ISSUE #245: Unicode and sidebar patch with overflowing lines bleeds into sidebar itself
I've noticed that unicode characters and sidebar patch sometimes leads to overflowing characters into the sidebar patch.  See screenshot below.  Beyond reporting this, I've done nothing to investigate why.
unicode-oops

### PR DESCRIPTION
**Pull request by gahr**
Closes #245

### CODE REVIEW

**Comment by gahr** [comment]
@neomutt/reviewers Please review

File: pager.c [hunk 1]
```diff
@@ -1339,7 +1339,7 @@ static int format_line (struct line_t **lineInfo, int n, unsigned char *buf,
     {
       if (col + 1 > wrap_cols)
 	break;
-      ++col;
+      col += k;
```
Comment (toogley) [inline pager.c]: `k` is used in a wide scope (100 lines) but its never really explained what it does.

Comment (gahr) [inline pager.c]: I'm still unsure whether this should be `col += wcwidth(replacement_char ());` Let me sleep over it some more :)

Comment (gahr) [inline pager.c]: @toogley a refactoring of pager.c could be in order - but let's do that elsewhere. It's a particular bug I'm trying to get caught here :)

Comment (toogley) [inline pager.c]: @gahr agreed.

**Pull request merged**